### ISSUE TITLE
[🐛] The sound of the ad video will continue to play after the ad is closed - interstitial ads

### ISSUE THREAD

**Issue by a1993azad** [invertase/react-native-google-mobile-ads#365]
### What happened?

When I close an interstitial ad sometimes the sound of the video does not stop with the ad closing, and it will continue to play until the end of the ad in the background
I tested it in IOS in production mode

### Platforms

iOS, but have not tested behavior on Android

### React Native Info

```Shell
info Fetching system and libraries information...
System:
    OS: macOS 12.6.3
    CPU: (4) x64 Intel(R) Core(TM) i5-3210M CPU @ 2.50GHz
    Memory: 185.67 MB / 16.00 GB
    Shell: 5.8.1 - /bin/zsh
  Binaries:
    Node: 16.16.0 - /usr/local/bin/node
    Yarn: 1.22.19 - /usr/local/bin/yarn
    npm: 8.11.0 - /usr/local/bin/npm
    Watchman: 2023.03.27.00 - /usr/local/bin/watchman
  Managers:
    CocoaPods: Not Found
  SDKs:
    iOS SDK:
      Platforms: DriverKit 22.2, iOS 16.2, macOS 13.1, tvOS 16.1, watchOS 9.1
    Android SDK: Not Found
  IDEs:
    Android Studio: 2022.1 AI-221.6008.13.2211.9619390
    Xcode: 14.2/14C18 - /usr/bin/xcodebuild
  Languages:
    Java: 11.0.17 - /usr/bin/javac
  npmPackages:
    @react-native-community/cli: Not Found
    react: Not Found
    react-native: Not Found
    react-native-macos: Not Found
  npmGlobalPackages:
    *react-native*: Not Found
```

### Are your using Typescript?

- [ ] My project is using Typescript

### package.json

```JSON
"@react-native-firebase/analytics": "^17.1.0",
    "@react-native-firebase/app": "^17.1.0",
    "@react-native-firebase/auth": "^17.1.0",
    "@react-native-firebase/crashlytics": "^17.1.0",
    "@react-native-firebase/messaging": "^17.1.0",
    "@react-native-firebase/perf": "^17.1.0",
    "@react-native-firebase/remote-config": "^17.1.0",
    "@react-native-google-signin/google-signin": "^9.0.2",
    "@react-navigation/native": "^6.1.4",
  "react-native-google-mobile-ads": "^9.1.1",
"react": "18.1.0",
    "react-native": "0.70.5",
"expo": "~47.0.3",
```

### app.json

```JSON
{
}
```

**Comment by ildfreelancer** [comment]
We get the same issues on few devices on production
But I cannot reproduce it on my local devices
It is really hard to fix
Can someone give me some hint why it happens?

**Comment by gyl-jer** [comment]
The issue has not been fixed, about 0.5-2% of our iOS users are experiencing this issue. We cannot reproduce it, but several users have sent videos and I can confirm the background music from the ad is playing - even if no ads are shown.

**Comment by sametcl2** [comment]
This happened me too
